### ISSUE TITLE
symmetric_rescaling() does not work for csc

### ISSUE THREAD

**Issue by lukeolson** [pyamg/pyamg#166]
per ben southworth: symmetric_rescaling(A) scales A like AD^{-1} if A is a csc-matrix, as opposed to D^{-1/2}AD^{-1/2}

**Comment by lukeolson** [comment]
Appears correct:

```
import numpy as np
from scipy.sparse import spdiags
from pyamg.util.utils import symmetric_rescaling
n=5
e = np.ones((n,1)).ravel()
data = [ -1*e, 2*e, -1*e ]
A = spdiags(data,[-1,0,1],n,n).tocsc()
Ds, Dsi, DAD = symmetric_rescaling(A)
print(DAD.todense())
```

```
[[ 1.  -0.5  0.   0.   0. ]
 [-0.5  1.  -0.5  0.   0. ]
 [ 0.  -0.5  1.  -0.5  0. ]
 [ 0.   0.  -0.5  1.  -0.5]
 [ 0.   0.   0.  -0.5  1. ]]
```

**Comment by lukeolson** [comment]
```
import numpy as np
from scipy.sparse import spdiags
from pyamg.util.utils import symmetric_rescaling
n=5
e = np.ones((n,1)).ravel()
data = [ -1*e, 2*e, -1*e ]
A = spdiags(data,[-1,0,1],n,n).tocsc()
A[:,2] *= 3
A[2,:] *=3
Ds, Dsi, DAD = symmetric_rescaling(A)
print(DAD.todense())
```

```
[[ 1.         -0.5         0.          0.          0.        ]
 [-0.5         1.         -0.16666667  0.          0.        ]
 [ 0.         -1.5         1.         -1.5         0.        ]
 [ 0.          0.         -0.16666667  1.         -0.5       ]
 [ 0.          0.          0.         -0.5         1.        ]]
```

**Comment by lukeolson** [comment]
fixed in 15c55fd5fcd8fa9ea98ce2762fd360eab14d7a7a
